releases.go

```diff
@@ -207,8 +213,13 @@ func showReleaseInfo(c *gin.Context) {
 		respErr(c, fmt.Errorf("bad info %s, release info only support all/hooks/manifest/notes/values", info))
 		return
 	}
-
-	actionConfig, err := actionConfigInit(namespace)
+	var options KubeInfoCommon
```

Comment:
there only `kube_context`, may be no need `KubeInfoCommon` struct, we can get context by `c.Query("kube_context")`, it's easy then BindJSON from request body

Reply:
Is good that we add the new parameter into the API? In fact, I'd like to keep the API in the same format which like the original. I mean without no more parameters, but that way you said above is more really easy way. Is there exist any standards of the API  we can follow?

Reply:
I saw the document on `w3.org`, some text like 

`By convention, when GET method is used, all information required to identify the resource is encoded in the URI. There is no convention in HTTP/1.1 for a safe interaction (e.g., retrieval) where the client supplies data to the server in an HTTP entity body rather than in the query part of a URI. This means that for safe operations, URIs may be long. The case of large parameters to a safe operation is not directly addressed by HTTP as it is presently deployed. A QUERY or "safe POST" or "GET with BODY" method has been discussed (e.g., at the December 1996 IETF meeting) but no consensus has emerged.`

So, let's add the parameter into the URL when we use the GET method